### ISSUE TITLE
`Open Visual Stimulator` Multi paths

### ISSUE THREAD

**Issue by AbdulrhmnGhanem** [kitspace/kitspace-v2#359]
https://github.com/eulerlab/open-visual-stimulator

The project is imported successfully 

Though clicking on any project card doesn't redirect to the multi-project; its links are broken. The mutlti-project names includes `/` which should be escaped!
```yaml
# kitsapce.yaml
multi:
    boards/mosfet_circuit:
      ...
    boards/lcr_add-on:
      ...
    boards/ssr_board:
      ...
```
![screencapture-abdo-dev-staging-kitspace-dev-tester-open-visual-stimulator-2022-05-23-22_37_36](https://user-images.githubusercontent.com/37152329/169902045-7a0a314c-73e5-4d76-8ac3-cbf5e9203007.png)

**Comment by AbdulrhmnGhanem** [comment]
Sulgify the names

**Comment by AbdulrhmnGhanem** [comment]
Closed via #402
